codespell_lib/data/dictionary.txt

```diff
@@ -24918,6 +24944,10 @@ tipically->typically
 titel->title
 titels->titles
 titile->title
+tittle->title
+tittled->titled
+tittles->titles
```

Comment:
It seems tittle and tittles should be in dictionary-rare please.

Reply:
Done 0c85d78